`isMuted`-Error-Widget is very verbose

Seeing a lot of 
```

[2025-04-16][15:12:55.470577][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.590981][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.591888][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.592229][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.592505][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.592893][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.593428][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.652147][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.652511][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.652642][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.652761][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.652950][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.653211][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.653587][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.704479][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.704849][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.704983][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.705099][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.705302][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.705565][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.705991][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.742255][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.742606][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.742736][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.742848][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.743028][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.743279][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.743649][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.772004][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null 
[2025-04-16][15:12:55.772337][a3::chat-item::mute-icon-widget][ERROR] Failed to load isMuted Mode not set null
```

which seems unnecessary. Refs https://github.com/acterglobal/a3-meta/issues/921

--- a/native/acter/src/api/room.rs
+++ b/native/acter/src/api/room.rs
@@ -1039,14 +1039,14 @@ impl Room {
 
     pub async fn notification_mode(&self) -> Result<String> {
         let room = self.room.clone();
-        RUNTIME
+        Ok(RUNTIME
             .spawn(async move {
                 room.user_defined_notification_mode()
                     .await
                     .map(|x| room_notification_mode_name(&x))
+                    .unwrap_or("none".to_string())
             })
-            .await?
-            .context("Mode not set")
+            .await?)
     }
 
     pub async fn default_notification_mode(&self) -> Result<String> {

